frontend/src/pages/ProgressionChecker/GridView/GridModal/styles.jsx

```diff
@@ -0,0 +1,47 @@
+import { Modal, Typography } from "antd";
+import styled from "styled-components";
+import GRID from "../styles";
+
+const { Title } = Typography;
+
+const ModalHeader = styled.div`
+  padding-top: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+`;
+
+const ModalTitle = styled(Title)`
+  && {
```

Comment:
what does && do here?

Reply:
ran into some trouble from antd so i read this https://github.com/styled-components/styled-components/issues/1816

seems like !important does the trick anyway - so will change it :)